tests/functional/testplan/graph/graph_example.py

```diff
@@ -0,0 +1,152 @@
+"""
```

Comment:
This file is a repetition of examples/Assertions/Graph/test_plan.py  
Should it be here?

Reply:
It has a slightly different purpose (documentation vs functional testing). However since they are so similar we could probably just re-use the example in the functional test.